### PR TITLE
Refactor all css text style

### PR DESCRIPTION
**Pull request by GolfGrab**

### CODE REVIEW

File: src/modules/menu/components/RecommendedFoodCard.tsx
Comment (JiMeow) [inline src/modules/menu/components/RecommendedFoodCard.tsx:43]: What does this line do? Does we need it?

**Review by JiMeow — COMMENTED** [review]
Almost every thing look great. good job :),
Need to recheck style with design in figma again before approve

File: src/modules/menu/components/RecommendedFoodCard.tsx
Comment (GolfGrab) [reply src/modules/menu/components/RecommendedFoodCard.tsx:43]: https://css-tricks.com/aspect-ratio-boxes/

Comment (GolfGrab) [reply src/modules/menu/components/RecommendedFoodCard.tsx:43]: it is css trick to get 1/1 aspect ratio 
component css padding-top : x%  this x is percentage of width
so this component wil tall as same as it width

Comment (JiMeow) [reply src/modules/menu/components/RecommendedFoodCard.tsx:43]: Got it, good job

**Review by JiMeow — APPROVED** [review]
All code good 👍👍 (not check design yet)

**Comment by JiMeow** [comment]
> All code good 👍👍 (not check design yet)

1. Padding in each page should be equal (now not equal)
* Home -> 20px
* Basket -> 20px
* My Orders -> 24px
2. font weight (bold text) not eqaul to figma design

**Review by minza55113151 — DISMISSED** [review]
GOD JOB. wait for Jirat review again.

File: src/modules/menu/components/RecommendedCategory.tsx
Comment (JiMeow) [inline src/modules/menu/components/RecommendedCategory.tsx:17]: dont need this line

File: src/modules/common/components/Typography.tsx
Comment (JiMeow) [inline src/modules/common/components/Typography.tsx:19]: optional: may move this line to bottom of the file

File: src/modules/order/components/OrderSummaryPrice.tsx
Comment (JiMeow) [inline src/modules/order/components/OrderSummaryPrice.tsx:40]: Optional: Can we make helper to change size of Text by give if prop?

**Review by JiMeow — APPROVED** [review]
Nice 10x developer, good job

**Pull request merged**
